system/security/shared/authmap.cpp

```diff
@@ -123,7 +123,7 @@ bool CAuthMap::shareWithManager(ISecManager& manager, IEspSecureContext* secureC
         ISecResourceList* curlist = (ISecResourceList*)m_resourcelists.item(x).list();
         if(curlist == NULL)
             continue;
-        ISecUser* usr = NULL;
+        Owned<ISecUser> usr = manager.createUser(nullptr, secureContext);
```

Comment:
Is it going to cause problems passing null for the user?

Reply:
No. It will not cause problems. @RussWhitehead please double confirm.

Reply:
@wangkx @ghalliday Not be a problem